chttpd.c

```diff
@@ -0,0 +1,122 @@
+// Copyright 2023 Vahid Mardani
```

Comment:
License header missing:

```
// Copyright 2023 Vahid Mardani
/*
 * This file is part of Carrow.
 *  Carrow is free software: you can redistribute it and/or modify it under 
 *  the terms of the GNU General Public License as published by the Free 
 *  Software Foundation, either version 3 of the License, or (at your option) 
 *  any later version.
 *  
 *  Carrow is distributed in the hope that it will be useful, but WITHOUT ANY 
 *  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS 
 *  FOR A PARTICULAR PURPOSE. See the GNU General Public License for more 
 *  details.
 *  
 *  You should have received a copy of the GNU General Public License along 
 *  with Carrow. If not, see <https://www.gnu.org/licenses/>. 
 *  
 *  Author: Vahid Mardani <vahid.mardani@gmail.com>
 */
```

Please replace any `Carrow` with `chttpd` in the license header.

Reply:
I suppose you were precisely implying that we name it `chttpd` with a small `c`.
Therefore, from now on it must be written `chttpd` in all passages, except when it is placed in the first word of sentences(conforming to English rules).

All good?